packages/genesys-spark-components/src/components/beta/gux-time-zone-picker/tests/gux-time-zone-picker.e2e.ts

```diff
@@ -43,7 +43,7 @@ describe('gux-time-zone-picker-beta', () => {
     await openWithClick(page);
     await a11yCheck(page, axeExclusions);
     let visibleItems = await unfilteredOptions(page);
-    expect(visibleItems.length).toBe(588);
+    expect(visibleItems.length).toBe(583);
```

Comment:
This must be related to how a puppeteer selector has changed in the different version.